src/components/CardsList.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import { BrowserRouter, Route, Link } from 'react-router-dom';
+import CardsListItem from './CardsListItem';
+import CardsItem from './CardsItem'
+
+export default class CardsList extends Component {
+  state = {
+    cards: null
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/cards.json')
```

Comment:
For future.
Move it anywhere to actions. And add anything for working with environment. F.E 
Development: 'localhost:3000',
Production: 'slothcards.com',
staging: 'stage.slothcards.com'
for not changing everytime host for different environments